### PR TITLE
Fix deprecations on django 1.11

### PR DESCRIPTION
**Pull request by bertonha**
This also makes it incompatible with django < 1.8

### CODE REVIEW

**Comment by pedrospdc** [comment]
pls merge

**Comment by erickwilder** [comment]
You need to update version dependencies on `setup.py` as well

**Comment by erickwilder** [comment]
Since this is removing support for older Python/Django versions I would bump a major version here https://github.com/vandersonmota/model_mommy/blob/master/model_mommy/__init__.py

Suggestion: `2.0.0`

**Comment by bertonha** [comment]
@erickwilder I will leave this for the @vandersonmota, since the bump of version is out of scope of this PR

**Comment by erickwilder** [comment]
Fair enough 🎉  LGTM then

File: .travis.yml [hunk 1]
```diff
@@ -1,34 +1,13 @@
...
 python:
-  - "2.6"
   - "2.7"
-  - "3.4"
   - "3.5"
+  - "3.6"
 env:
-  - DJANGO=django16
-  - DJANGO=django17
-  - DJANGO=django18
```
Comment (ownaginatious) [inline .travis.yml]: Given that Django 1.8 is an LTS release (scheduled for deprecation in [April 2018](https://www.djangoproject.com/weblog/2015/dec/01/django-19-released/)), I think support should be kept for it. The only thing that I think would need to be changed is something like this around the `rel` vs `remote_field` changes.

e.g.

```python
...
remote_field = 'remote_field' if django.VERSION >= (1, 9) else 'rel'
if isinstance(field, OneToOneField) and getattr(field, remote_field).parent_link:
              return True
...
```

**Comment by ownaginatious** [comment]
@bertonha cool, LGTM 👍 @vandersonmota please take a look :)

**Comment by vandersonmota** [comment]
@berinhard can you take a look at this?

**Comment by berinhard** [comment]
I'm travelling on vacations @vandersonmota with limited access to internet =/

**Comment by vandersonmota** [comment]
@bertonha can you make this PR against the `development` branch?

**Comment by vandersonmota** [comment]
Ah, BTW @bertonha Thanks!

**Comment by ownaginatious** [comment]
@vandersonmota seems @bertonha has changed the base to `development` per your feedback :)

**Comment by pedrospdc** [comment]
Can we get this merged? This PR is over a month already :(

**Comment by vandersonmota** [comment]
@pedrospdc sorry, i was dedicating free time for myself and family lately. I'll make sure this doesn't happen in the future.

**Comment by vandersonmota** [comment]
thanks @bertonha

**Comment by ownaginatious** [comment]
@vandersonmota thanks for merging. Could you push a new version to PyPI too?

**Comment by ownaginatious** [comment]
@berinhard @vandersonmota PyPI push?

**Pull request merged**